streaming_kafka_consumer/strategy_factory.py

```diff
@@ -114,7 +107,6 @@ def create(
                 max_batch_time=self.__max_batch_time,
                 input_block_size=self.__input_block_size,
                 output_block_size=self.__output_block_size,
-                metrics=MetricsWrapper(self.__metrics, "process"),
```

Comment:
Yeah sorry my comment is outdated, I figured it's not worth tampering with the metric names people give us and removed the MetricsWrapper from the library entirely.

Reply:
OK so this will cause changes to our current metric names, then?

Reply:
It will cause a change to one of our metric names:
`snuba.consumer.process.batches_in_progress` will become `snuba.consumer.batches_in_progress`.

Since this code is becoming agnostic of Snuba, I'm not convinced it's particularly worthwhile to include the `process` in there (and need the MetricsWrapper in the library to handle it).

Reply:
Got it. OK makes sense.